velox/core/QueryConfig.h

```diff
@@ -188,6 +188,9 @@ class QueryConfig {
   static constexpr const char* kSpillableReservationGrowthPct =
       "spillable_reservation_growth_pct";
 
+  /// If true, array_agg() aggregation function will ignore nulls in the input.
+  static constexpr const char* kArrayAggIgnoreNulls = "array_agg_ignore_nulls";
```

Comment:
Let's also update documentation for array_agg itself to explain that its behavior depends on a config.